examples/js/loaders/GLTFLoader.js

```diff
@@ -2654,6 +2678,8 @@ THREE.GLTFLoader = ( function () {
 
 			if ( nodeDef.extras ) node.userData = nodeDef.extras;
 
```

Comment:
It's true that any GLTF object can have extensions right? Material is most immediately useful for us, but just 'scene' would also be useful.

Reply:
Awesome! 'mesh' may also be useful, but not immediate for me.